chapters/division_of_work/divisiton_of_work.tex

```diff
@@ -0,0 +1,56 @@
+\chapter{Division of work}\label{ch:division_of_work}
+This chapter describes the division of work on the application and the document.
+The division of work on the application is presented in \autoref{tab:division_of_work_on_the_application} and the division of work on the document is presented in \autoref{tab:division_of_work_on_the_document}.
+It is important to note that all parts of the project were worked on by both authors.
+Each line in both the application source code and the document written by one of the authors was reviewed by the other author in a pull request.
+Moreover both authors made bug fixes and small changes to all parts of the project.
+These tables only show who the main author of each part was.
+
+\begin{table}[h]
+    \centering
+    \begin{tabular}{|l|l|}
+    \hline
+    Task     & Author        \\ \hline
+    Implementation of the game logic & Both \\
+    Implementation of the graphics engine & Both \\
+    Implementation of the spherical geometry & Aleksy Bałaziński      \\
+    Implementation of the hyperbolic geometry & Aleksy Bałaziński      \\
+    Implementation of the physics engine & Aleksy Bałaziński      \\
```

Comment:
Maybe "Integration of the physics engine with the rest of the system" or something like that

Reply:
maybe also add a separate row with "Terrain editing" with "Both" we've spent quite some time on it